src/smart-components/workflow/edit-workflow-info-modal.js

```diff
@@ -69,7 +84,7 @@ const EditWorkflowInfoModal = ({
                   id="save-edit-workflow-info"
                   variant="primary"
                   type="submit"
-                  isDisabled={ isFetching }
+                  isDisabled={ isFetching || !formValid(formData) }
```

Comment:
Pf4 forms should have the submit button enabled even if there are validation errors. If that is the case, cliking on the submit should focus the first invalid field